codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/recursivestructtype.java

```diff
@@ -276,6 +291,14 @@ public final Builder recursiveList(RecursiveStructType... recursiveList) {
             return this;
         }
 
+        @Override
+        @SafeVarargs
+        public final Builder recursiveList(Consumer<Builder>... recursiveList) {
+            recursiveList(Arrays.asList(recursiveList).stream().map(c -> RecursiveStructType.builder().apply(c).build())
```

Comment:
Can you use Stream.of here? Possibly more efficient (not sure how it's implemented) but is a bit more concise too.

Reply:
Updated!